SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1802,6 +1802,27 @@ void USpatialNetDriver::ProcessRPC(AActor* Actor, UObject* SubObject, UFunction*
 {
 	using namespace SpatialNetDriverPrivate;
 
+	FString WorkerId = [&]() -> FString {
+		if (World == nullptr)
+		{
+			return "(no world)";
+		}
+
+		const UGameInstance* GameInstance = World->GetGameInstance();
+		if (GameInstance == nullptr)
+		{
+			return "(no game instance)";
+		}
+
+		return GameInstance->GetSpatialWorkerId();
+	}();
+	FString FunctionName = Function->GetName();
+
+	if (FunctionName == "ServerSetEnabled")
+	{
+		UE_LOG(LogTemp, Warning, TEXT("[%s] ProcessRPC %s %s"), *WorkerId, *Function->GetName(), *Actor->GetFullName());
```

Comment:
if we hit this, the RPC will actually make it through to spatialos and will be sent off